integrations/dbt/snowflake/dbt/adapters/openlineage_snowflake/schema/bq-error-run-facet.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "definitions": {
+    "BigQueryErrorRunFacet": {
```

Comment:
Are these facets correct? Looks like they're defined for `BigQuery`.

Reply:
Removed. Relics of code copying...